### PR TITLE
Refactor: rename ComputedMapping to DispatchTable

### PR DESCRIPTION
**Pull request by t0yv0**
Noticing we were getting confused between this mapping and GetMapping call. A new name. Mechanical rename PR, no behavior changes.

### CODE REVIEW

**Review by iwahbe — APPROVED** [review]
LGTM assuming we pass local unit tests. We need to release a new muxer v0.0.2 to get downstream tests to pass.

**Comment by t0yv0** [comment]
Oh I see, yes.

**Comment by t0yv0** [comment]
I'll tag it after I merge this.

**Pull request merged**
